src/Administration/Login/TokenService/TokenService.php

```diff
@@ -0,0 +1,32 @@
+<?php declare(strict_types=1);
+
+namespace Shopware\Administration\Login\TokenService;
+
+use Shopware\Administration\LoginConfig\ConfigBuilder\LoginConfigService;
+use Symfony\Contracts\HttpClient\HttpClientInterface;
+
+final class TokenService
```

Comment:
Just my POV: the name `TokenService` isn't very descriptive, as we are dealing with a lot of different tokens in multiple places. I think we should either put them all under a descriptive namespace or rename them to something like `OAuthTokenService` or `OIDCTokenService` (as the scope is hardcoded to `openid`), or even something like `ExternalTokenService` to make it clear that this is dealing with an external OAuth server